frontend/src/bundles/friends/pages/pages.ts

```diff
@@ -0,0 +1 @@
+export { Friends } from './friends/friends.js';
```

Comment:
I think you don't need that reexport here. We have just one page in friends - friends.tsx

Reply:
@valery-chumak  Thank you for reviewing. I have already fixed it.